nservicebus/upgrades/release-policy.md

```diff
@@ -79,3 +80,45 @@ Since all critical issues will be back-ported, choose to upgrade when convenient
 ### Major
 
 Since a new `major` version will contain breaking changes and will likely require modifications to the consuming code. Because of the extent of code changes in a major version, it is recommendation that a full regression test of the system is done.
+
+
+### Release quality
+
+The release cycle consists of the following quality stages:
+
+ * Alpha
+ * Beta (Optional)
+ * Release Candidate (Optional)
+ * Stable
+
+
+### Alpha
+
+ * Binaries are considered unstable.
+ * APIs might change without notice.
+ * Packages are used for internal development, updating docs, etc.
+
+
+### Beta
+
+ * The component is feature complete.
+ * Likely to contain a number of known or unknown bugs.
+ * Performance and stability testing has not been fully completed.
+ * **No** "go live" license is provided and production use is **not supported**.
+ * All public APIs should be stable but may change based on feedback from consumers.
+
+
+#### Open and closed beta
+
+ * Closed beta versions are released to a restricted group customers by invitation.
+ * Open beta versions are public and target the entire user base.
+
+
+### Release Candidate (RC)
+
+A Release Candidate is a version intended to be a final product, which is ready to release unless significant bugs emerge. In this stage of stabilization, all features have been designed, coded and tested through one or more beta cycles and has no known critical bugs. There could still be source code changes to fix defects, changes to documentation and data files, and peripheral code for test cases or utilities.
+
+
+### Release To Market (RTM)
+
+This is the stable production release.
```

Comment:
@SimonCropp - After thinking about it some more, if this is the release to market, support is implied. So I didn't add it. Do you think I need to add it?